index.html

```diff
@@ -7,7 +7,7 @@
 		<link rel="stylesheet" href="./node_modules/prosemirror-view/style/prosemirror.css">
 		<link rel="stylesheet" href="style/main.css">
 		<link rel="stylesheet" href="style/math.css">
-		<script src="./src/index.ts"></script>
+		<script src="./index.ts"></script>
```

Comment:
I have moved the demo to the main folder. and then started to replace the index file with exports. More work necessary there.